### DISCUSSION TITLE
Is runtime compilation available?

### DISCUSSION THREAD

**Discussion by PedroGameDesigner** [ephread/inkgd#86] [Q&A] [answered]
Looking at the "Running your Skin" documentation I saw that you can set up to compile .ink files on runtime, but I wasn't able to find the equivalent functions on inkgd.
Is that functionality available?

**Comment by francoisdlt** [comment]
do you mean "on export" ? or are you looking into compiling dynamic ink at runtime ?

**Reply by PedroGameDesigner** [reply]
I wanted to compile ink files at runtime.
I'm working on a project that, if possible, I want players to mod it with their own small story snippets

**Reply by francoisdlt** [reply, marked as answer]
I dont think that is supported, you'll need to write your own code for that and ship an inklecate binary to compile the JSON whenever needed.

**Reply by PedroGameDesigner** [reply]
Thanks.
I'm might look into how to do that, but I'm probably not going to implement it as my project is supposed to be very code-light.
